website/content/en/status/report-2025-04-2025-06/foundation-sta.adoc

```diff
@@ -0,0 +1,104 @@
+Two members of the Foundation project staff will be present, along with two representatives from Bitergia who delivered the GrimoireLab implementation for this project.
+(Members of the FreeBSD Project Source Management team were not available to attend.)
+
+Progress is being made to reduce technical debt by creating an link:https://github.com/linimon/patchQA[automated method] for evaluating patches (code improvements) attached to existing pull requests for source and ports trees to see whether they are still relevant, and applying them if they are.
+This tool is in beta.
+
+==== Work Package B: Zero Trust Builds
+
+This work package intends to improve tooling and processes to support Zero Trust Builds of FreeBSD by extending the current components to enable the project to build release artifacts (package sets, ISO images, etc.) without requiring any special privilege.
+
+The detailed scope was co-created with core@, srcmgr@, secteam@. Work items are as follows:
+
+* Must
+** No-root for all source release build cases/artifacts (in progress)
+** Src artifacts to build reproducibly (in progress)
+** Formalize and document make world and release.sh (in progress)
+* Should
+** Remove privilege from orchestration tooling (not started)
+** Move build scripts into the public repository (not started)
+* Could
+** Environment Standardization (not started)
+** Use enclaves (not started)
+** Ports to build reproducibly (not started)
+** CI to verify reproducibility (in progress)
+** Documentation to allow 3rd parties to confirm reproducibility (not started)
+
+===== Work Package C: CI/CD Automation
+
+This work package intends to improve CI/CD automation to streamline software delivery and operations for new and existing software by modernizing and securitizing the existing CI/CD system and extending it to cover the third party packages in the FreeBSD Ports Collection.
+
+The detailed scope was co-created with core@, srcmgr@, portmgr@, doceng@.
+
+* Must
+** Improve quality of incoming commits (completed)
+** Pre-merge CI (completed)
+** Environment Metadata (not started)
+** Extend CI to the Ports tree (in progress)
+** CI Threat Model (not started)
+** CI Management Process (in progress)
+** Documentation (not started)
+* Should
+** 3rd-party Interoperability (in progress)
+** Automated analysis in tests (in progress)
+** Test Case Management (not started)
+* Could
+** Granular Debugging (not started)
+
+===== Work Package D: Ports and Packages security improvements
+
+This work package intends to modernize and extend security controls in the FreeBSD Ports and Package Collection by: Migrating from our VuXML Vulnerability Database to OSV or similar contemporary format; developing a package audit backend and server to reliably fetch vulnerability data from global agency databases in any format (JSON - NIST) and produce insight and; improving CI tooling for FreeBSD Ports.
```

Comment:
I think "... produce insight and; ..." should instead be "... produce insight; and ..." instead